src/org/thoughtcrime/securesms/loki/redesign/activities/HomeActivity.kt

```diff
@@ -243,21 +246,44 @@ class HomeActivity : PassphraseRequiredActionBarActivity, ConversationClickListe
                             apiDatabase.removeLastDeletionServerID(publicChat.channel, publicChat.server)
                             ApplicationContext.getInstance(activity).lokiPublicChatAPI!!.leave(publicChat.channel, publicChat.server)
                         }
+
```

Comment:
The rest of the code in this file uses comments to separate chunks of code rather than empty lines (which are used to separate members). It's good practice to keep in line with the rest of the file you're working in so that things stay consistent.